test/integration/lib/json/bridge_status.json

```diff
@@ -0,0 +1,445 @@
+      "modelid": "LCT007",
+      "manufacturername": "Philips",
+      "uniqueid": "00:17:88:01:10:41:87:79-0b",
+      "swversion": "66014919"
+    },
+    "2": {
+      "state": {
+        "on": true,
+        "bri": 252,
+        "hue": 0,
+        "sat": 0,
+        "effect": "none",
+        "xy": [
+        0.5186,
+        0.2357
+        ],
+        "ct": 233,
+        "alert": "none",
+        "colormode": "xy",
+        "reachable": false
+      },
+      "type": "Extended color light",
+      "name": "Hue color lamp 2",
+      "modelid": "LCT007",
+      "manufacturername": "Philips",
+      "uniqueid": "00:17:88:01:10:56:b6:21-0b",
+      "swversion": ""
+    },
+    "3": {
+      "state": {
+        "on": true,
+        "bri": 254,
+        "hue": 14910,
+        "sat": 144,
+        "effect": "none",
+        "xy": [
+        0.4596,
+        0.4105
+        ],
+        "ct": 370,
+        "alert": "none",
+        "colormode": "ct",
+        "reachable": false
+      },
+      "type": "Extended color light",
+      "name": "Hue color lamp 3",
+      "modelid": "LCT007",
+      "manufacturername": "Philips",
+      "uniqueid": "00:17:88:01:10:56:ba:92-0b",
+      "swversion": "66014919"
+    }
+  },
+  "groups": {},
+  "config": {
+    "name": "Philips hue",
+    "zigbeechannel": 11,
+    "bridgeid": "001788FFFE230B96",
+    "mac": "00:17:88:23:0b:96",
+    "dhcp": true,
+    "ipaddress": "192.168.1.140",
+    "netmask": "255.255.255.0",
+    "gateway": "192.168.1.1",
+    "proxyaddress": "none",
+    "proxyport": 0,
+    "UTC": "2016-02-29T21:44:47",
+    "localtime": "2016-02-29T16:44:47",
+    "timezone": "America/Toronto",
+    "modelid": "BSB002",
+    "swversion": "01030262",
+    "apiversion": "1.11.0",
+    "swupdate": {
+      "updatestate": 0,
+      "checkforupdate": false,
+      "devicetypes": {
+        "bridge": false,
+        "lights": [],
+        "sensors": []
+      },
+      "url": "",
+      "text": "",
+      "notify": false
+    },
+    "linkbutton": false,
+    "portalservices": true,
+    "portalconnection": "connected",
+    "portalstate": {
+      "signedon": true,
+      "incoming": true,
+      "outgoing": true,
+      "communication": "disconnected"
+    },
+    "factorynew": false,
+    "replacesbridgeid": null,
+    "backup": {
+      "status": "idle",
+      "errorcode": 0
+    },
+    "whitelist": {
+      "2aebf17e5cbd09c3712b116b68325934": {
+        "last use date": "2016-02-29T04:18:37",
+        "create date": "2016-02-26T01:44:55",
+        "name": "Hue#LGE Nexus 4"
+      },
+      "foxbox-abfa90c57d538145": {
+        "last use date": "2016-02-29T21:44:47",
+        "create date": "2016-02-26T01:44:55",
+        "name": "foxbox_hub"
+      },
+      "24955b892a1ccc434a83e3974a77f337": {
+        "last use date": "2016-02-26T06:48:37",
+        "create date": "2016-02-26T06:20:02",
+        "name": "my_hue_app#iphone peter"
+      }
+    }
+  },
+  "schedules": {},
+  "scenes": {
```

Comment:
I wonder if we could make the scene part smaller, and just have 2 scenes in it. What do you think?